observability/src/main/java/io/grpc/observability/logging/GcpLogSink.java

```diff
@@ -68,6 +74,19 @@ public GcpLogSink(String destinationProjectId) {
 
   @Override
   public void write(GrpcLogRecord logProto) {
+    this.write(logProto, null, null);
+  }
+
+  /**
+   * Writes logs to GCP Cloud Logging.
+   *
+   * @param logProto gRPC logging proto containing the message to be logged
+   * @param locationTags Key-value pairs to identify GCP compute resource
+   * @param customTags User specified key-value pairs to be attached to every log record
+   */
+  @Override
+  public void write(GrpcLogRecord logProto, Map<String, String> locationTags,
```

Comment:
As mentioned elsewhere, locationTags and customTags are "constant" through out. Can we just make them members of `GcpLogSink` so they don't need to be passed via `write` every time?